### DISCUSSION TITLE
Conciliação duvidas

### DISCUSSION THREAD

**Discussion by jeancarlosc39** [bacen/pix-api#341] [Q&A]
Estou fazendo as conciliações mas estou vendo que não tenho 
Status do movimento | Enum: emissão, exclusão comandada, exclusão automática, exclusão por portabilidade, expiração, recebimento, devolução integral, devolução parcial  e sempre me retorna ativo
Me retorna assim o movimento
 [nome_pagador] => JEAN CARLO GONCALVES                    
                            [cnpj_pagador] => 00337638950
                            [tipo_documento] => dinamico
                            [id_documento] => 29e2c122-9538-4353-b5c7-d183c9e695f6
                            [id_transacao] => aBpVL06SZ5VHbY8kmzytclZJ7AhhvHEqIuh
                            [id_lancamento] => 73c14810-c898-4b0c-bf1b-4a9b14a88v38
                            [valor_original] => 32.35
                            [valor_final] => 32.35
                            [valor_pago] => 32.35
                            [valor_desconto] => 0
                            [valor_juros] => 0
                            [valor_multa] => 0
                            [status] => ativo

Bom no exemplo da saída do itau me retorna mais informações como data de vencimento e vi que tem um body 

onde a vários campos opcionais na saida o que achei estranho. 

-- | -- | -- | --

data": [{
                                            "nome_pagador": "Joao Siqueira",
                                            "cnpj_pagador": "44333094000109",
                                            "tipo_documento": "estático",
                                            "id_documento": "b1ff5cc0-8a9c-497e-b983-738904c23389",
                                            "id_transacao": "b1ff5cc0-8a9c-497e-b983-738904c23389",
                                            "data_vencimento": "2020-08-04T12:00:00",
                                            "valor_original": 1000.00,
                                            "valor_final": 1000.00,
                                            "valor_pago": 1000.00,
                                            "valor_devolvido": 1000.00,
                                            "valor_desconto": 10.00,
                                            "valor_juros": 20.00,
                                            "valor_multa": 20.00,
                                            "valor_tarifa": 4.0,
                                            "status": "emissão"

**Comment by renatofrota** [comment]
@jeancarlosc39 ,

aparentemente, sua mensagem (e outras que você já postou nesse repositório) tem dados sensíveis.

Se for o caso, recomendo editar (e apagar a versão original da publicação) pela interface do GitHub.
